core/templates/services/compute-graph.service.ts

```diff
@@ -25,6 +25,7 @@ import { States } from 'domain/exploration/StatesObjectFactory';
 export interface GraphLink {
   source: string;
   target: string;
+  linkProperty?: string;
```

Comment:
Why is this optional?

Reply:
because sometimes we are not passing property `linkProperty` to the variable of type GraphLink in whole codebase and some time we are passing linkProperty a value.

Reply:
? removed in PR https://github.com/oppia/oppia/pull/15859, PTAL